site/app/controllers/student/TeamController.php

```diff
@@ -422,4 +422,8 @@ public function showPage($gradeable_id) {
         $users_seeking_team = $this->core->getQueries()->getUsersSeekingTeamByGradeableId($gradeable_id);
         $this->core->getOutput()->renderOutput(array('submission', 'Team'), 'showTeamPage', $gradeable, $team, $teams, $lock, $users_seeking_team);
     }
+    public function getTeamIds($id, $gradeable_id) {
```

Comment:
You should move this query to the controller it belongs to. This function doesn't need to be here if it's only used in a completely different controller.